governance/polls/Ratification Poll - MIP Amendment Subproposal (MIP102c2-SP9) - June 12, 2023.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP9) - June 12, 2023
+summary: This subproposal incorporates the Bug Bounty Programa (originally MIP64) into MIP107.
+discussion_link: https://forum.makerdao.com/t/mip102c2-sp9-mip-amendment-subproposal/20640
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 10000 }
+            ]
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-06-12T16:00:00
+end_date: 2023-06-26T16:00:00
+---
+# Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP9) - June 12, 2023
+
+The Arbitration Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for fourteen days beginning on Monday, June 12 at 16:00 UTC.
```

Comment:
```suggestion
The Governance Facilitator has placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for fourteen days beginning on Monday, June 12 at 16:00 UTC.
```

Reply:
Should the last link here go to the monthly governance cycle page rather than the weekly?